internal/clients/image_builder/image_client.go

```diff
@@ -56,7 +60,8 @@ func (c *ImageBuilderClient) fetchImageStatus(ctx context.Context, composeID str
 		ctxval.Logger(ctx).Warn().Msgf("Image builder replied with unexpected status while fetching image status: %v", statusCode)
 		return nil, ClientErr
 	}
-	err = verifyImage(resp.JSON200)
+	ctxval.Logger(ctx).Debug().Msgf("Fetching image status was finished %+v\n", resp.JSON200.ImageStatus.UploadStatus)
```

Comment:
This is something you used for debugging, not sure how useful is this from the operational perspective.